### PR TITLE
feat: remove aggrid and update memberships tables

### PR DESCRIPTION
**Pull request by pyphilia**
Remove ag-grid and update membership table.

### CODE REVIEW

File: src/components/item/sharing/InvitationsTable.tsx [hunk 1]
```diff
@@ -45,142 +40,91 @@ const InvitationsTable = ({
...
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {invitations.map((row) => (
+            <TableRow
+              key={row.id}
+              sx={{ '&:last-child td, &:last-child th': { border: 0 } }}
+              data-cy={buildInvitationTableRowId(row.id)}
+            >
+              <TableCell component="th" scope="row">
```
Comment (spaenleh) [inline src/components/item/sharing/InvitationsTable.tsx:97]: Why override the component to use a `th` ?

**Review by spaenleh — APPROVED** [review]
Looks good ! Thank you for your work on this !

File: src/components/item/sharing/TableRowPermission.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    (!allowDowngrade && permission === PermissionLevel.Admin) ? (
+    <Typography noWrap>{permission}</Typography>
+  ) : (
+    <ItemMembershipSelect
+      value={permission}
+      showLabel={false}
+      allowDowngrade={allowDowngrade}
+      onChange={onChangePermission}
+    />
+  );
```
[the remark this answers is not in the document]
Comment (ReidyT) [reply src/components/item/sharing/TableRowPermission.tsx:36]: I agree with Basile.

**Review by ReidyT — APPROVED** [review]
Thanks for this PR, LGTM! 🧑‍🚀

**Pull request merged**